spring-pulsar-docs/src/main/antora/antora.yml

```diff
@@ -10,3 +10,7 @@ ext:
       local: true
     scan:
       dir: spring-pulsar-docs/build/generated-antora-resources
+asciidoc:
```

Comment:
Ahhh ok. This is component version metadata vs. playbook global metadata, correct?

Reply:
I just noticed your descriptions in the commit. Thanks for that. It answers this question.